corehq/apps/translations/app_translations/upload_form.py

```diff
@@ -48,12 +48,21 @@ def _get_xform(self):
             return self.form.wrapped_xform()
 
     def _get_itext(self):
+        """
+        find the bucket node that holds all translations.
+        it has a bunch of nodes, one for each lang, which then
+        has translations for all labels as a child node, example
+        <itext>
+        <translation lang="en" default="">
+         <text id="name-label">
+           <value>Name2</value>
+           <value form="image">image_path</value>
+         </text>
+        </translation>
+        </itext>
+        """
         if self.xform:
-            try:
-                return self.xform.itext_node
-            except XFormException:
```

Comment:
I realize you already had this conversation in https://github.com/dimagi/commcare-hq/pull/24075#discussion_r276982390 so apologies for arriving late to the party, but I wanted to verify: do you know exactly what happens now if you do bulk translations on an app with a totally blank form? I'm guessing a 500, since blank forms don't have an itext node.

Reply:
No worries. I think this one is still open to discussion so all good.
Thats a valid point.
So this would mean that if a user adds a new form without adding any questions, its possible that there is no itext node yet added by Vellum and hence this would break. Let me test that.

Reply:
Thanks @orangejenny, that case did break so i have reverted the commit with a more specific comment on why we should let this exception go.
I was trying to make a change such that we should not even worry about updating the form if there are no rows in the sheet, that way this exception would be a real raise but i think that goes out of the context of this PR

Reply:
Cool, thanks for investigating. Did you possibly miss pushing your changes? I don't see a revert.